package.json

```diff
@@ -49,6 +51,8 @@
     "predist": "npm run eslint",
     "prepack": "fixpack && npm run dist",
     "pretest": "npm run dist && npm run build-test-addon-xpi",
-    "test": "export FIREFOX_BINARY=firefox && XPI_NAME=test-addon/test-addon.xpi mocha test"
+    "test": "export FIREFOX_BINARY=firefox && XPI_NAME=test-addon/test-addon.xpi mocha test",
```

Comment:
This command doesn't work unfortunately since most people's `firefox` alias will point to Release Firefox, which doesn't allow unsigned legacy addons to install. I would omit the path variable (or we can at least set it to `nightly`) for now.

If you are comfortable providing instructions in the README for how devs can edit their bash profile to set `FIREFOX_BINARY`, that'd be a great addition. I made an [issue](https://github.com/mozilla/shield-studies-addon-template/issues/54) in the template repo.

Reply:
Leaving this as is, since it is not in the change for this PR.